colors/multicolor.md

```diff
@@ -3,25 +3,20 @@
 > You are remembered for the rules you break.
 > <cite>Douglas MacArthur</cite>
 
-Multicolor cards go beyond a single color. They have more than one color in their energy cost, so you
-need to have ways to generate both colors of energy to play them. Their effects are often aligned with more
-than one color and would not be possible with just a single color.
+Multicolor cards go beyond a single color. They have more than one color in their energy cost, so you need to have ways to generate both colors of energy to play them. Their effects are often aligned with more than one color and would not be possible with just a single color.
 
 ![Unbound Radical](https://s3.amazonaws.com/assets1.orbsccg.com/prod/cards/art/59X.jpg)
 
-When energized, multicolor cards do not contain enough of a single color to generate energy of any color, so they
-can only generate colorless energy. This means that you will need to compliment multiccolor cards with monocolor cards
-in order to generate the energy you need for your multicolor deck.
+When energized, multicolor cards do not contain enough of a single color to generate energy of any color, so they can only generate colorless energy. This means that you will need to compliment multicolor cards with monocolor cards in order to generate the energy you need for your multicolor deck.
+
+[//]: # 'curious why is that, maybe some kind of other things like enters tapped? if that is something possible?'
```

Comment:
It's because it'd be too easy to just fill your deck with multicolor cards if they all made energy of any color. They'd be too strong.

Reply:
maybe energize to 1 color of the pair